### DISCUSSION TITLE
cdnjs use feedback

### DISCUSSION THREAD

**Discussion by Half-Sugar-Code** [cdnjs/cdnjs#14154] [General] [answered]
Hi, I'm using (https://cdnjs.com/) to retrieve resources and I'm seeing some errors when typing in the search box. Is this correct?

**Comment by MattIPv4** [comment, marked as answer]
👋 Could you share more about what you mean by "I'm seeing some unpredictable errors"? Are you seeing search results you don't expect, or are you seeing the search fail to load, or something else?

**Reply by Half-Sugar-Code** [reply]
When you enter content in the input box, the content will appear jumping

**Reply by MattIPv4** [reply]
Could you share a screen recording of the issue you're seeing? When I type in the search box I see no jumping, it just loads the search results as each character is typed.

**Reply by Half-Sugar-Code** [reply]
I found that when my input method is transferred to Chinese, there will be such a problem.
